id,name,raw,category,processed,solution,target,version,targetnew,label
5676,6170,"2022-10-19 11:36:05,019 | Failed at Play [wintools_complete_install_verify] **********
2022-10-19 11:36:05,019 | TASK [wintools_complete_install_verify][Configure VM CDROM to 'iso'] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_Server_2022_64/ansible-vsphere-gos-validation/common/vm_configure_cdrom.yml:14
fatal: [localhost]: FAILED! => Connection control operation failed for disk 'sata0:0'.",,timestamp failed at play win tools complete install verify timestamp task win tools complete install verify configure vm cdrom to iso task path home worker workspace ansible cycle windows server number ansible vsphere gos validation common vm configure cdrom yml number fatal localhost failed connection control operation failed for disk sata number number,deepdive,targetvm,202210200116,machine,relevant
5671,6181,"2022-10-19 16:47:36,019 | Failed at Play [deploy_vmwarephoton_ova] *******************
2022-10-19 16:47:36,019 | TASK [deploy_vmwarephoton_ova][Check VMware Tools is running and collects guest IPv4 address successfully] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:51
fatal: [localhost]: FAILED! => It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
error message:
It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
2022-10-19 16:48:01,019 | TASK [deploy_vmwarephoton_ova][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
error message:
Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
2022-10-19 16:48:14,019 | TASK [deploy_vmwarephoton_ova][Fetch file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found
error message:
Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found",,timestamp failed at play deploy vmware photon ova timestamp task deploy vmware photon ova check vmware tools is running and collects guest ip v number address successfully task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address error message it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address timestamp task deploy vmware photon ova testing exit due to failure task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vmware photon ova error message exit testing when exit testing when fail is set to true in test case deploy vmware photon ova timestamp task deploy vmware photon ova fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found error message guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found,retry,nimbus,202210200116,environment,relevant
5672,6185,"2022-10-19 19:54:37,019 | Failed at Play [deploy_vm_ovf] *****************************
2022-10-19 19:54:37,019 | TASK [deploy_vm_ovf][Deploy VM from ovf template] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/ovf_deploy.yml:4
exception in /vmware_deploy_ovf.py when run in /request.py when http_error_default
fatal: [localhost]: FAILED! => HTTP Error 500: Internal Server Error Problem validating OVF import spec: Line 98: Invalid value 'hostonly' for element 'Connection'.
2022-10-19 19:54:38,019 | TASK [deploy_vm_ovf][Testing exit due to failure] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_ovf",,timestamp failed at play deploy vm ovf timestamp task deploy vm ovf deploy vm from ovf template task path home worker workspace ansible windows ms template main ansible vsphere gos validation common ovf deploy yml number exception in vmware deploy ovf python when run in request python when http error default fatal localhost failed http error number internal server error problem validating ovf import spec line number invalid value host only for element connection timestamp task deploy vm ovf testing exit due to failure task path home worker workspace ansible windows ms template main ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm ovf,deepdive,testbed,202210200116,component,relevant
5673,6157,"2022-10-19 07:48:02,019 | Failed at Play [secureboot_enable_disable] *****************
2022-10-19 07:48:02,019 | TASK [secureboot_enable_disable][Check VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP address] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36
fatal: [localhost]: FAILED! => Failed to get VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP Address",,timestamp failed at play secure boot enable disable timestamp task secure boot enable disable check vm u os server v number pvscsi vmxnet number efi ip address task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm u os server v number pvscsi vmxnet number efi ip address,deepdive,targetvm,202210200116,machine,relevant
5674,6168,"2022-10-19 11:55:38,019 | Failed at Play [memory_hot_add_basic] **********************
2022-10-19 11:55:38,019 | TASK [memory_hot_add_basic][Skip testcase: memory_hot_add_basic, reason: Blocked] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/skip_test_case.yml:20
fatal: [localhost]: FAILED! => Test case 'memory_hot_add_basic' is blocked because memory hotadd test value list is empty",,timestamp failed at play memory hot add basic timestamp task memory hot add basic skip test case memory hot add basic reason blocked task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common skip test case yml number fatal localhost failed test case memory hot add basic is blocked because memory hot add test value list is empty,deepdive,testbed,202210200116,component,relevant
5675,6142,"2022-10-19 04:42:38,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 04:42:38,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Wait for message 'Autoinstall is completed.' appear in VM log serial-20221019032827.log] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35
fatal: [localhost]: FAILED! => {
    ""attempts"": 720,
    ""censored"": ""the output has been hidden due to the fact that 'no_log: true' was specified for this result"",
    ""changed"": false
}
2022-10-19 04:42:48,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Fetch file /tmp/cloud-init_2022-10-19-04-42-40.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Invalid guest login for user root",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task deploy vm efi para virtual vmxnet number fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed invalid guest login for user root,deepdive,targetvm,202210200116,machine,relevant
5677,6152,"2022-10-19 07:44:53,019 | Failed at Play [deploy_vm_efi_ide_e1000e] ******************
2022-10-19 07:44:53,019 | TASK [deploy_vm_efi_ide_e1000e][Datastore file operation] **
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:44:54,019 | TASK [deploy_vm_efi_ide_e1000e][Get specified property info for VM 'test_vm'] 
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm",,timestamp failed at play deploy vm efi ide timestamp task deploy vm efi ide data store file operation task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi ide get specified property info for vm test vm task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm,deepdive,testbed,202210200116,component,relevant
5678,6158,"2022-10-19 07:57:24,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 07:57:24,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Datastore file operation] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:57:25,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Get specified property info for VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263'] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number data store file operation task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi para virtual vmxnet number get specified property info for vm u os server v number pvscsi vmxnet number efi number task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm u os server v number pvscsi vmxnet number efi number,deepdive,testbed,202210200116,component,relevant
5676,6170,"2022-10-19 11:36:05,019 | Failed at Play [wintools_complete_install_verify] **********
2022-10-19 11:36:05,019 | TASK [wintools_complete_install_verify][Configure VM CDROM to 'iso'] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_Server_2022_64/ansible-vsphere-gos-validation/common/vm_configure_cdrom.yml:14
fatal: [localhost]: FAILED! => Connection control operation failed for disk 'sata0:0'.",,timestamp failed at play win tools complete install verify timestamp task win tools complete install verify configure vm cdrom to iso task path home worker workspace ansible cycle windows server number ansible vsphere gos validation common vm configure cdrom yml number fatal localhost failed connection control operation failed for disk sata number number,deepdive,targetvm,202210200116,environment,irrelevant
5676,6170,"2022-10-19 11:36:05,019 | Failed at Play [wintools_complete_install_verify] **********
2022-10-19 11:36:05,019 | TASK [wintools_complete_install_verify][Configure VM CDROM to 'iso'] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_Server_2022_64/ansible-vsphere-gos-validation/common/vm_configure_cdrom.yml:14
fatal: [localhost]: FAILED! => Connection control operation failed for disk 'sata0:0'.",,timestamp failed at play win tools complete install verify timestamp task win tools complete install verify configure vm cdrom to iso task path home worker workspace ansible cycle windows server number ansible vsphere gos validation common vm configure cdrom yml number fatal localhost failed connection control operation failed for disk sata number number,deepdive,targetvm,202210200116,component,irrelevant
5676,6170,"2022-10-19 11:36:05,019 | Failed at Play [wintools_complete_install_verify] **********
2022-10-19 11:36:05,019 | TASK [wintools_complete_install_verify][Configure VM CDROM to 'iso'] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_Server_2022_64/ansible-vsphere-gos-validation/common/vm_configure_cdrom.yml:14
fatal: [localhost]: FAILED! => Connection control operation failed for disk 'sata0:0'.",,timestamp failed at play win tools complete install verify timestamp task win tools complete install verify configure vm cdrom to iso task path home worker workspace ansible cycle windows server number ansible vsphere gos validation common vm configure cdrom yml number fatal localhost failed connection control operation failed for disk sata number number,deepdive,targetvm,202210200116,infrastructure,irrelevant
5676,6170,"2022-10-19 11:36:05,019 | Failed at Play [wintools_complete_install_verify] **********
2022-10-19 11:36:05,019 | TASK [wintools_complete_install_verify][Configure VM CDROM to 'iso'] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_Server_2022_64/ansible-vsphere-gos-validation/common/vm_configure_cdrom.yml:14
fatal: [localhost]: FAILED! => Connection control operation failed for disk 'sata0:0'.",,timestamp failed at play win tools complete install verify timestamp task win tools complete install verify configure vm cdrom to iso task path home worker workspace ansible cycle windows server number ansible vsphere gos validation common vm configure cdrom yml number fatal localhost failed connection control operation failed for disk sata number number,deepdive,targetvm,202210200116,user,irrelevant
5676,6170,"2022-10-19 11:36:05,019 | Failed at Play [wintools_complete_install_verify] **********
2022-10-19 11:36:05,019 | TASK [wintools_complete_install_verify][Configure VM CDROM to 'iso'] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_Server_2022_64/ansible-vsphere-gos-validation/common/vm_configure_cdrom.yml:14
fatal: [localhost]: FAILED! => Connection control operation failed for disk 'sata0:0'.",,timestamp failed at play win tools complete install verify timestamp task win tools complete install verify configure vm cdrom to iso task path home worker workspace ansible cycle windows server number ansible vsphere gos validation common vm configure cdrom yml number fatal localhost failed connection control operation failed for disk sata number number,deepdive,targetvm,202210200116,test,irrelevant
5676,6170,"2022-10-19 11:36:05,019 | Failed at Play [wintools_complete_install_verify] **********
2022-10-19 11:36:05,019 | TASK [wintools_complete_install_verify][Configure VM CDROM to 'iso'] 
task path: /home/worker/workspace/Ansible_Cycle_Windows_Server_2022_64/ansible-vsphere-gos-validation/common/vm_configure_cdrom.yml:14
fatal: [localhost]: FAILED! => Connection control operation failed for disk 'sata0:0'.",,timestamp failed at play win tools complete install verify timestamp task win tools complete install verify configure vm cdrom to iso task path home worker workspace ansible cycle windows server number ansible vsphere gos validation common vm configure cdrom yml number fatal localhost failed connection control operation failed for disk sata number number,deepdive,targetvm,202210200116,product,irrelevant
5671,6181,"2022-10-19 16:47:36,019 | Failed at Play [deploy_vmwarephoton_ova] *******************
2022-10-19 16:47:36,019 | TASK [deploy_vmwarephoton_ova][Check VMware Tools is running and collects guest IPv4 address successfully] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:51
fatal: [localhost]: FAILED! => It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
error message:
It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
2022-10-19 16:48:01,019 | TASK [deploy_vmwarephoton_ova][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
error message:
Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
2022-10-19 16:48:14,019 | TASK [deploy_vmwarephoton_ova][Fetch file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found
error message:
Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found",,timestamp failed at play deploy vmware photon ova timestamp task deploy vmware photon ova check vmware tools is running and collects guest ip v number address successfully task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address error message it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address timestamp task deploy vmware photon ova testing exit due to failure task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vmware photon ova error message exit testing when exit testing when fail is set to true in test case deploy vmware photon ova timestamp task deploy vmware photon ova fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found error message guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found,retry,nimbus,202210200116,component,irrelevant
5671,6181,"2022-10-19 16:47:36,019 | Failed at Play [deploy_vmwarephoton_ova] *******************
2022-10-19 16:47:36,019 | TASK [deploy_vmwarephoton_ova][Check VMware Tools is running and collects guest IPv4 address successfully] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:51
fatal: [localhost]: FAILED! => It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
error message:
It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
2022-10-19 16:48:01,019 | TASK [deploy_vmwarephoton_ova][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
error message:
Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
2022-10-19 16:48:14,019 | TASK [deploy_vmwarephoton_ova][Fetch file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found
error message:
Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found",,timestamp failed at play deploy vmware photon ova timestamp task deploy vmware photon ova check vmware tools is running and collects guest ip v number address successfully task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address error message it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address timestamp task deploy vmware photon ova testing exit due to failure task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vmware photon ova error message exit testing when exit testing when fail is set to true in test case deploy vmware photon ova timestamp task deploy vmware photon ova fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found error message guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found,retry,nimbus,202210200116,infrastructure,irrelevant
5671,6181,"2022-10-19 16:47:36,019 | Failed at Play [deploy_vmwarephoton_ova] *******************
2022-10-19 16:47:36,019 | TASK [deploy_vmwarephoton_ova][Check VMware Tools is running and collects guest IPv4 address successfully] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:51
fatal: [localhost]: FAILED! => It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
error message:
It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
2022-10-19 16:48:01,019 | TASK [deploy_vmwarephoton_ova][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
error message:
Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
2022-10-19 16:48:14,019 | TASK [deploy_vmwarephoton_ova][Fetch file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found
error message:
Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found",,timestamp failed at play deploy vmware photon ova timestamp task deploy vmware photon ova check vmware tools is running and collects guest ip v number address successfully task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address error message it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address timestamp task deploy vmware photon ova testing exit due to failure task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vmware photon ova error message exit testing when exit testing when fail is set to true in test case deploy vmware photon ova timestamp task deploy vmware photon ova fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found error message guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found,retry,nimbus,202210200116,user,irrelevant
5671,6181,"2022-10-19 16:47:36,019 | Failed at Play [deploy_vmwarephoton_ova] *******************
2022-10-19 16:47:36,019 | TASK [deploy_vmwarephoton_ova][Check VMware Tools is running and collects guest IPv4 address successfully] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:51
fatal: [localhost]: FAILED! => It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
error message:
It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
2022-10-19 16:48:01,019 | TASK [deploy_vmwarephoton_ova][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
error message:
Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
2022-10-19 16:48:14,019 | TASK [deploy_vmwarephoton_ova][Fetch file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found
error message:
Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found",,timestamp failed at play deploy vmware photon ova timestamp task deploy vmware photon ova check vmware tools is running and collects guest ip v number address successfully task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address error message it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address timestamp task deploy vmware photon ova testing exit due to failure task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vmware photon ova error message exit testing when exit testing when fail is set to true in test case deploy vmware photon ova timestamp task deploy vmware photon ova fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found error message guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found,retry,nimbus,202210200116,test,irrelevant
5671,6181,"2022-10-19 16:47:36,019 | Failed at Play [deploy_vmwarephoton_ova] *******************
2022-10-19 16:47:36,019 | TASK [deploy_vmwarephoton_ova][Check VMware Tools is running and collects guest IPv4 address successfully] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:51
fatal: [localhost]: FAILED! => It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
error message:
It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
2022-10-19 16:48:01,019 | TASK [deploy_vmwarephoton_ova][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
error message:
Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
2022-10-19 16:48:14,019 | TASK [deploy_vmwarephoton_ova][Fetch file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found
error message:
Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found",,timestamp failed at play deploy vmware photon ova timestamp task deploy vmware photon ova check vmware tools is running and collects guest ip v number address successfully task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address error message it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address timestamp task deploy vmware photon ova testing exit due to failure task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vmware photon ova error message exit testing when exit testing when fail is set to true in test case deploy vmware photon ova timestamp task deploy vmware photon ova fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found error message guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found,retry,nimbus,202210200116,machine,irrelevant
5671,6181,"2022-10-19 16:47:36,019 | Failed at Play [deploy_vmwarephoton_ova] *******************
2022-10-19 16:47:36,019 | TASK [deploy_vmwarephoton_ova][Check VMware Tools is running and collects guest IPv4 address successfully] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:51
fatal: [localhost]: FAILED! => It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
error message:
It's timed out for VMware Tools collecting guest IPv4 address in 300 seconds. Current VMware Tools running status is 'guestToolsRunning' and guest IP addresses are '['fe80::250:56ff:feac:eb4e']'.
2022-10-19 16:48:01,019 | TASK [deploy_vmwarephoton_ova][Testing exit due to failure] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
error message:
Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vmwarephoton_ova
2022-10-19 16:48:14,019 | TASK [deploy_vmwarephoton_ova][Fetch file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Regression_Photon_3.0_OVA/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found
error message:
Guest file /tmp/cloud-init_2022-10-19-16-48-05.tar.gz does not exist : File /tmp/cloud-init_2022-10-19-16-48-05.tar.gz was not found",,timestamp failed at play deploy vmware photon ova timestamp task deploy vmware photon ova check vmware tools is running and collects guest ip v number address successfully task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm wait guest ip yml number fatal localhost failed it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address error message it s timed out for vmware tools collecting guest ip v number address in number seconds current vmware tools running status is guest tools running and guest ip addresses are ip address timestamp task deploy vmware photon ova testing exit due to failure task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vmware photon ova error message exit testing when exit testing when fail is set to true in test case deploy vmware photon ova timestamp task deploy vmware photon ova fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible regression photon number ova ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found error message guest file tmp cloud init timestamp number tar gz does not exist file tmp cloud init timestamp number tar gz was not found,retry,nimbus,202210200116,product,irrelevant
5672,6185,"2022-10-19 19:54:37,019 | Failed at Play [deploy_vm_ovf] *****************************
2022-10-19 19:54:37,019 | TASK [deploy_vm_ovf][Deploy VM from ovf template] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/ovf_deploy.yml:4
exception in /vmware_deploy_ovf.py when run in /request.py when http_error_default
fatal: [localhost]: FAILED! => HTTP Error 500: Internal Server Error Problem validating OVF import spec: Line 98: Invalid value 'hostonly' for element 'Connection'.
2022-10-19 19:54:38,019 | TASK [deploy_vm_ovf][Testing exit due to failure] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_ovf",,timestamp failed at play deploy vm ovf timestamp task deploy vm ovf deploy vm from ovf template task path home worker workspace ansible windows ms template main ansible vsphere gos validation common ovf deploy yml number exception in vmware deploy ovf python when run in request python when http error default fatal localhost failed http error number internal server error problem validating ovf import spec line number invalid value host only for element connection timestamp task deploy vm ovf testing exit due to failure task path home worker workspace ansible windows ms template main ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm ovf,deepdive,testbed,202210200116,environment,irrelevant
5672,6185,"2022-10-19 19:54:37,019 | Failed at Play [deploy_vm_ovf] *****************************
2022-10-19 19:54:37,019 | TASK [deploy_vm_ovf][Deploy VM from ovf template] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/ovf_deploy.yml:4
exception in /vmware_deploy_ovf.py when run in /request.py when http_error_default
fatal: [localhost]: FAILED! => HTTP Error 500: Internal Server Error Problem validating OVF import spec: Line 98: Invalid value 'hostonly' for element 'Connection'.
2022-10-19 19:54:38,019 | TASK [deploy_vm_ovf][Testing exit due to failure] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_ovf",,timestamp failed at play deploy vm ovf timestamp task deploy vm ovf deploy vm from ovf template task path home worker workspace ansible windows ms template main ansible vsphere gos validation common ovf deploy yml number exception in vmware deploy ovf python when run in request python when http error default fatal localhost failed http error number internal server error problem validating ovf import spec line number invalid value host only for element connection timestamp task deploy vm ovf testing exit due to failure task path home worker workspace ansible windows ms template main ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm ovf,deepdive,testbed,202210200116,infrastructure,irrelevant
5672,6185,"2022-10-19 19:54:37,019 | Failed at Play [deploy_vm_ovf] *****************************
2022-10-19 19:54:37,019 | TASK [deploy_vm_ovf][Deploy VM from ovf template] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/ovf_deploy.yml:4
exception in /vmware_deploy_ovf.py when run in /request.py when http_error_default
fatal: [localhost]: FAILED! => HTTP Error 500: Internal Server Error Problem validating OVF import spec: Line 98: Invalid value 'hostonly' for element 'Connection'.
2022-10-19 19:54:38,019 | TASK [deploy_vm_ovf][Testing exit due to failure] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_ovf",,timestamp failed at play deploy vm ovf timestamp task deploy vm ovf deploy vm from ovf template task path home worker workspace ansible windows ms template main ansible vsphere gos validation common ovf deploy yml number exception in vmware deploy ovf python when run in request python when http error default fatal localhost failed http error number internal server error problem validating ovf import spec line number invalid value host only for element connection timestamp task deploy vm ovf testing exit due to failure task path home worker workspace ansible windows ms template main ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm ovf,deepdive,testbed,202210200116,user,irrelevant
5672,6185,"2022-10-19 19:54:37,019 | Failed at Play [deploy_vm_ovf] *****************************
2022-10-19 19:54:37,019 | TASK [deploy_vm_ovf][Deploy VM from ovf template] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/ovf_deploy.yml:4
exception in /vmware_deploy_ovf.py when run in /request.py when http_error_default
fatal: [localhost]: FAILED! => HTTP Error 500: Internal Server Error Problem validating OVF import spec: Line 98: Invalid value 'hostonly' for element 'Connection'.
2022-10-19 19:54:38,019 | TASK [deploy_vm_ovf][Testing exit due to failure] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_ovf",,timestamp failed at play deploy vm ovf timestamp task deploy vm ovf deploy vm from ovf template task path home worker workspace ansible windows ms template main ansible vsphere gos validation common ovf deploy yml number exception in vmware deploy ovf python when run in request python when http error default fatal localhost failed http error number internal server error problem validating ovf import spec line number invalid value host only for element connection timestamp task deploy vm ovf testing exit due to failure task path home worker workspace ansible windows ms template main ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm ovf,deepdive,testbed,202210200116,test,irrelevant
5672,6185,"2022-10-19 19:54:37,019 | Failed at Play [deploy_vm_ovf] *****************************
2022-10-19 19:54:37,019 | TASK [deploy_vm_ovf][Deploy VM from ovf template] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/ovf_deploy.yml:4
exception in /vmware_deploy_ovf.py when run in /request.py when http_error_default
fatal: [localhost]: FAILED! => HTTP Error 500: Internal Server Error Problem validating OVF import spec: Line 98: Invalid value 'hostonly' for element 'Connection'.
2022-10-19 19:54:38,019 | TASK [deploy_vm_ovf][Testing exit due to failure] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_ovf",,timestamp failed at play deploy vm ovf timestamp task deploy vm ovf deploy vm from ovf template task path home worker workspace ansible windows ms template main ansible vsphere gos validation common ovf deploy yml number exception in vmware deploy ovf python when run in request python when http error default fatal localhost failed http error number internal server error problem validating ovf import spec line number invalid value host only for element connection timestamp task deploy vm ovf testing exit due to failure task path home worker workspace ansible windows ms template main ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm ovf,deepdive,testbed,202210200116,machine,irrelevant
5672,6185,"2022-10-19 19:54:37,019 | Failed at Play [deploy_vm_ovf] *****************************
2022-10-19 19:54:37,019 | TASK [deploy_vm_ovf][Deploy VM from ovf template] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/ovf_deploy.yml:4
exception in /vmware_deploy_ovf.py when run in /request.py when http_error_default
fatal: [localhost]: FAILED! => HTTP Error 500: Internal Server Error Problem validating OVF import spec: Line 98: Invalid value 'hostonly' for element 'Connection'.
2022-10-19 19:54:38,019 | TASK [deploy_vm_ovf][Testing exit due to failure] **********
task path: /home/worker/workspace/Ansible_Windows_MS_Template_MAIN/ansible-vsphere-gos-validation/common/test_rescue.yml:59
fatal: [localhost]: FAILED! => Exit testing when 'exit_testing_when_fail' is set to True in test case deploy_vm_ovf",,timestamp failed at play deploy vm ovf timestamp task deploy vm ovf deploy vm from ovf template task path home worker workspace ansible windows ms template main ansible vsphere gos validation common ovf deploy yml number exception in vmware deploy ovf python when run in request python when http error default fatal localhost failed http error number internal server error problem validating ovf import spec line number invalid value host only for element connection timestamp task deploy vm ovf testing exit due to failure task path home worker workspace ansible windows ms template main ansible vsphere gos validation common test rescue yml number fatal localhost failed exit testing when exit testing when fail is set to true in test case deploy vm ovf,deepdive,testbed,202210200116,product,irrelevant
5673,6157,"2022-10-19 07:48:02,019 | Failed at Play [secureboot_enable_disable] *****************
2022-10-19 07:48:02,019 | TASK [secureboot_enable_disable][Check VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP address] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36
fatal: [localhost]: FAILED! => Failed to get VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP Address",,timestamp failed at play secure boot enable disable timestamp task secure boot enable disable check vm u os server v number pvscsi vmxnet number efi ip address task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm u os server v number pvscsi vmxnet number efi ip address,deepdive,targetvm,202210200116,environment,irrelevant
5673,6157,"2022-10-19 07:48:02,019 | Failed at Play [secureboot_enable_disable] *****************
2022-10-19 07:48:02,019 | TASK [secureboot_enable_disable][Check VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP address] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36
fatal: [localhost]: FAILED! => Failed to get VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP Address",,timestamp failed at play secure boot enable disable timestamp task secure boot enable disable check vm u os server v number pvscsi vmxnet number efi ip address task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm u os server v number pvscsi vmxnet number efi ip address,deepdive,targetvm,202210200116,component,irrelevant
5673,6157,"2022-10-19 07:48:02,019 | Failed at Play [secureboot_enable_disable] *****************
2022-10-19 07:48:02,019 | TASK [secureboot_enable_disable][Check VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP address] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36
fatal: [localhost]: FAILED! => Failed to get VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP Address",,timestamp failed at play secure boot enable disable timestamp task secure boot enable disable check vm u os server v number pvscsi vmxnet number efi ip address task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm u os server v number pvscsi vmxnet number efi ip address,deepdive,targetvm,202210200116,infrastructure,irrelevant
5673,6157,"2022-10-19 07:48:02,019 | Failed at Play [secureboot_enable_disable] *****************
2022-10-19 07:48:02,019 | TASK [secureboot_enable_disable][Check VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP address] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36
fatal: [localhost]: FAILED! => Failed to get VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP Address",,timestamp failed at play secure boot enable disable timestamp task secure boot enable disable check vm u os server v number pvscsi vmxnet number efi ip address task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm u os server v number pvscsi vmxnet number efi ip address,deepdive,targetvm,202210200116,user,irrelevant
5673,6157,"2022-10-19 07:48:02,019 | Failed at Play [secureboot_enable_disable] *****************
2022-10-19 07:48:02,019 | TASK [secureboot_enable_disable][Check VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP address] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36
fatal: [localhost]: FAILED! => Failed to get VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP Address",,timestamp failed at play secure boot enable disable timestamp task secure boot enable disable check vm u os server v number pvscsi vmxnet number efi ip address task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm u os server v number pvscsi vmxnet number efi ip address,deepdive,targetvm,202210200116,test,irrelevant
5673,6157,"2022-10-19 07:48:02,019 | Failed at Play [secureboot_enable_disable] *****************
2022-10-19 07:48:02,019 | TASK [secureboot_enable_disable][Check VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP address] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_ip.yml:36
fatal: [localhost]: FAILED! => Failed to get VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17' IP Address",,timestamp failed at play secure boot enable disable timestamp task secure boot enable disable check vm u os server v number pvscsi vmxnet number efi ip address task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get ip yml number fatal localhost failed failed to get vm u os server v number pvscsi vmxnet number efi ip address,deepdive,targetvm,202210200116,product,irrelevant
5674,6168,"2022-10-19 11:55:38,019 | Failed at Play [memory_hot_add_basic] **********************
2022-10-19 11:55:38,019 | TASK [memory_hot_add_basic][Skip testcase: memory_hot_add_basic, reason: Blocked] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/skip_test_case.yml:20
fatal: [localhost]: FAILED! => Test case 'memory_hot_add_basic' is blocked because memory hotadd test value list is empty",,timestamp failed at play memory hot add basic timestamp task memory hot add basic skip test case memory hot add basic reason blocked task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common skip test case yml number fatal localhost failed test case memory hot add basic is blocked because memory hot add test value list is empty,deepdive,testbed,202210200116,environment,irrelevant
5674,6168,"2022-10-19 11:55:38,019 | Failed at Play [memory_hot_add_basic] **********************
2022-10-19 11:55:38,019 | TASK [memory_hot_add_basic][Skip testcase: memory_hot_add_basic, reason: Blocked] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/skip_test_case.yml:20
fatal: [localhost]: FAILED! => Test case 'memory_hot_add_basic' is blocked because memory hotadd test value list is empty",,timestamp failed at play memory hot add basic timestamp task memory hot add basic skip test case memory hot add basic reason blocked task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common skip test case yml number fatal localhost failed test case memory hot add basic is blocked because memory hot add test value list is empty,deepdive,testbed,202210200116,infrastructure,irrelevant
5674,6168,"2022-10-19 11:55:38,019 | Failed at Play [memory_hot_add_basic] **********************
2022-10-19 11:55:38,019 | TASK [memory_hot_add_basic][Skip testcase: memory_hot_add_basic, reason: Blocked] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/skip_test_case.yml:20
fatal: [localhost]: FAILED! => Test case 'memory_hot_add_basic' is blocked because memory hotadd test value list is empty",,timestamp failed at play memory hot add basic timestamp task memory hot add basic skip test case memory hot add basic reason blocked task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common skip test case yml number fatal localhost failed test case memory hot add basic is blocked because memory hot add test value list is empty,deepdive,testbed,202210200116,user,irrelevant
5674,6168,"2022-10-19 11:55:38,019 | Failed at Play [memory_hot_add_basic] **********************
2022-10-19 11:55:38,019 | TASK [memory_hot_add_basic][Skip testcase: memory_hot_add_basic, reason: Blocked] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/skip_test_case.yml:20
fatal: [localhost]: FAILED! => Test case 'memory_hot_add_basic' is blocked because memory hotadd test value list is empty",,timestamp failed at play memory hot add basic timestamp task memory hot add basic skip test case memory hot add basic reason blocked task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common skip test case yml number fatal localhost failed test case memory hot add basic is blocked because memory hot add test value list is empty,deepdive,testbed,202210200116,test,irrelevant
5674,6168,"2022-10-19 11:55:38,019 | Failed at Play [memory_hot_add_basic] **********************
2022-10-19 11:55:38,019 | TASK [memory_hot_add_basic][Skip testcase: memory_hot_add_basic, reason: Blocked] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/skip_test_case.yml:20
fatal: [localhost]: FAILED! => Test case 'memory_hot_add_basic' is blocked because memory hotadd test value list is empty",,timestamp failed at play memory hot add basic timestamp task memory hot add basic skip test case memory hot add basic reason blocked task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common skip test case yml number fatal localhost failed test case memory hot add basic is blocked because memory hot add test value list is empty,deepdive,testbed,202210200116,machine,irrelevant
5674,6168,"2022-10-19 11:55:38,019 | Failed at Play [memory_hot_add_basic] **********************
2022-10-19 11:55:38,019 | TASK [memory_hot_add_basic][Skip testcase: memory_hot_add_basic, reason: Blocked] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/skip_test_case.yml:20
fatal: [localhost]: FAILED! => Test case 'memory_hot_add_basic' is blocked because memory hotadd test value list is empty",,timestamp failed at play memory hot add basic timestamp task memory hot add basic skip test case memory hot add basic reason blocked task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common skip test case yml number fatal localhost failed test case memory hot add basic is blocked because memory hot add test value list is empty,deepdive,testbed,202210200116,product,irrelevant
5675,6142,"2022-10-19 04:42:38,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 04:42:38,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Wait for message 'Autoinstall is completed.' appear in VM log serial-20221019032827.log] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35
fatal: [localhost]: FAILED! => {
    ""attempts"": 720,
    ""censored"": ""the output has been hidden due to the fact that 'no_log: true' was specified for this result"",
    ""changed"": false
}
2022-10-19 04:42:48,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Fetch file /tmp/cloud-init_2022-10-19-04-42-40.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Invalid guest login for user root",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task deploy vm efi para virtual vmxnet number fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed invalid guest login for user root,deepdive,targetvm,202210200116,environment,irrelevant
5675,6142,"2022-10-19 04:42:38,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 04:42:38,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Wait for message 'Autoinstall is completed.' appear in VM log serial-20221019032827.log] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35
fatal: [localhost]: FAILED! => {
    ""attempts"": 720,
    ""censored"": ""the output has been hidden due to the fact that 'no_log: true' was specified for this result"",
    ""changed"": false
}
2022-10-19 04:42:48,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Fetch file /tmp/cloud-init_2022-10-19-04-42-40.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Invalid guest login for user root",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task deploy vm efi para virtual vmxnet number fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed invalid guest login for user root,deepdive,targetvm,202210200116,component,irrelevant
5675,6142,"2022-10-19 04:42:38,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 04:42:38,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Wait for message 'Autoinstall is completed.' appear in VM log serial-20221019032827.log] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35
fatal: [localhost]: FAILED! => {
    ""attempts"": 720,
    ""censored"": ""the output has been hidden due to the fact that 'no_log: true' was specified for this result"",
    ""changed"": false
}
2022-10-19 04:42:48,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Fetch file /tmp/cloud-init_2022-10-19-04-42-40.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Invalid guest login for user root",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task deploy vm efi para virtual vmxnet number fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed invalid guest login for user root,deepdive,targetvm,202210200116,infrastructure,irrelevant
5675,6142,"2022-10-19 04:42:38,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 04:42:38,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Wait for message 'Autoinstall is completed.' appear in VM log serial-20221019032827.log] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35
fatal: [localhost]: FAILED! => {
    ""attempts"": 720,
    ""censored"": ""the output has been hidden due to the fact that 'no_log: true' was specified for this result"",
    ""changed"": false
}
2022-10-19 04:42:48,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Fetch file /tmp/cloud-init_2022-10-19-04-42-40.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Invalid guest login for user root",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task deploy vm efi para virtual vmxnet number fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed invalid guest login for user root,deepdive,targetvm,202210200116,user,irrelevant
5675,6142,"2022-10-19 04:42:38,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 04:42:38,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Wait for message 'Autoinstall is completed.' appear in VM log serial-20221019032827.log] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35
fatal: [localhost]: FAILED! => {
    ""attempts"": 720,
    ""censored"": ""the output has been hidden due to the fact that 'no_log: true' was specified for this result"",
    ""changed"": false
}
2022-10-19 04:42:48,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Fetch file /tmp/cloud-init_2022-10-19-04-42-40.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Invalid guest login for user root",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task deploy vm efi para virtual vmxnet number fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed invalid guest login for user root,deepdive,targetvm,202210200116,test,irrelevant
5675,6142,"2022-10-19 04:42:38,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 04:42:38,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Wait for message 'Autoinstall is completed.' appear in VM log serial-20221019032827.log] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_wait_log_msg.yml:35
fatal: [localhost]: FAILED! => {
    ""attempts"": 720,
    ""censored"": ""the output has been hidden due to the fact that 'no_log: true' was specified for this result"",
    ""changed"": false
}
2022-10-19 04:42:48,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Fetch file /tmp/cloud-init_2022-10-19-04-42-40.tar.gz from VM guest] 
task path: /home/worker/workspace/Ansible_Cycle_Ubuntu_20.04_ISO/ansible-vsphere-gos-validation/common/vm_guest_file_operation.yml:91
exception in /vmware_guest_file_operation.py when fetch in /SoapAdapter.py when InvokeMethod
fatal: [localhost]: FAILED! => Invalid guest login for user root",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number wait for message auto install is completed appear in vm log serial timestamp log task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm wait log message yml number fatal localhost failed attempts number censored the output has been hidden due to the fact that no log true was specified for this result changed false timestamp task deploy vm efi para virtual vmxnet number fetch file tmp cloud init timestamp number tar gz from vm guest task path home worker workspace ansible cycle ubuntu number iso ansible vsphere gos validation common vm guest file operation yml number exception in vmware guest file operation python when fetch in soap adapter python when invoke method fatal localhost failed invalid guest login for user root,deepdive,targetvm,202210200116,product,irrelevant
5677,6152,"2022-10-19 07:44:53,019 | Failed at Play [deploy_vm_efi_ide_e1000e] ******************
2022-10-19 07:44:53,019 | TASK [deploy_vm_efi_ide_e1000e][Datastore file operation] **
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:44:54,019 | TASK [deploy_vm_efi_ide_e1000e][Get specified property info for VM 'test_vm'] 
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm",,timestamp failed at play deploy vm efi ide timestamp task deploy vm efi ide data store file operation task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi ide get specified property info for vm test vm task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm,deepdive,testbed,202210200116,environment,irrelevant
5677,6152,"2022-10-19 07:44:53,019 | Failed at Play [deploy_vm_efi_ide_e1000e] ******************
2022-10-19 07:44:53,019 | TASK [deploy_vm_efi_ide_e1000e][Datastore file operation] **
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:44:54,019 | TASK [deploy_vm_efi_ide_e1000e][Get specified property info for VM 'test_vm'] 
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm",,timestamp failed at play deploy vm efi ide timestamp task deploy vm efi ide data store file operation task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi ide get specified property info for vm test vm task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm,deepdive,testbed,202210200116,infrastructure,irrelevant
5677,6152,"2022-10-19 07:44:53,019 | Failed at Play [deploy_vm_efi_ide_e1000e] ******************
2022-10-19 07:44:53,019 | TASK [deploy_vm_efi_ide_e1000e][Datastore file operation] **
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:44:54,019 | TASK [deploy_vm_efi_ide_e1000e][Get specified property info for VM 'test_vm'] 
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm",,timestamp failed at play deploy vm efi ide timestamp task deploy vm efi ide data store file operation task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi ide get specified property info for vm test vm task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm,deepdive,testbed,202210200116,user,irrelevant
5677,6152,"2022-10-19 07:44:53,019 | Failed at Play [deploy_vm_efi_ide_e1000e] ******************
2022-10-19 07:44:53,019 | TASK [deploy_vm_efi_ide_e1000e][Datastore file operation] **
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:44:54,019 | TASK [deploy_vm_efi_ide_e1000e][Get specified property info for VM 'test_vm'] 
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm",,timestamp failed at play deploy vm efi ide timestamp task deploy vm efi ide data store file operation task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi ide get specified property info for vm test vm task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm,deepdive,testbed,202210200116,test,irrelevant
5677,6152,"2022-10-19 07:44:53,019 | Failed at Play [deploy_vm_efi_ide_e1000e] ******************
2022-10-19 07:44:53,019 | TASK [deploy_vm_efi_ide_e1000e][Datastore file operation] **
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:44:54,019 | TASK [deploy_vm_efi_ide_e1000e][Get specified property info for VM 'test_vm'] 
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm",,timestamp failed at play deploy vm efi ide timestamp task deploy vm efi ide data store file operation task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi ide get specified property info for vm test vm task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm,deepdive,testbed,202210200116,machine,irrelevant
5677,6152,"2022-10-19 07:44:53,019 | Failed at Play [deploy_vm_efi_ide_e1000e] ******************
2022-10-19 07:44:53,019 | TASK [deploy_vm_efi_ide_e1000e][Datastore file operation] **
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:44:54,019 | TASK [deploy_vm_efi_ide_e1000e][Get specified property info for VM 'test_vm'] 
task path: /home/worker/workspace/Ansible_UOS_Server_70U1_IDE_E1000E_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM test_vm",,timestamp failed at play deploy vm efi ide timestamp task deploy vm efi ide data store file operation task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi ide get specified property info for vm test vm task path home worker workspace ansible u os server ide efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm test vm,deepdive,testbed,202210200116,product,irrelevant
5678,6158,"2022-10-19 07:57:24,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 07:57:24,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Datastore file operation] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:57:25,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Get specified property info for VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263'] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number data store file operation task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi para virtual vmxnet number get specified property info for vm u os server v number pvscsi vmxnet number efi number task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm u os server v number pvscsi vmxnet number efi number,deepdive,testbed,202210200116,environment,irrelevant
5678,6158,"2022-10-19 07:57:24,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 07:57:24,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Datastore file operation] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:57:25,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Get specified property info for VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263'] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number data store file operation task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi para virtual vmxnet number get specified property info for vm u os server v number pvscsi vmxnet number efi number task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm u os server v number pvscsi vmxnet number efi number,deepdive,testbed,202210200116,infrastructure,irrelevant
5678,6158,"2022-10-19 07:57:24,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 07:57:24,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Datastore file operation] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:57:25,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Get specified property info for VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263'] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number data store file operation task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi para virtual vmxnet number get specified property info for vm u os server v number pvscsi vmxnet number efi number task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm u os server v number pvscsi vmxnet number efi number,deepdive,testbed,202210200116,user,irrelevant
5678,6158,"2022-10-19 07:57:24,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 07:57:24,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Datastore file operation] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:57:25,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Get specified property info for VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263'] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number data store file operation task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi para virtual vmxnet number get specified property info for vm u os server v number pvscsi vmxnet number efi number task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm u os server v number pvscsi vmxnet number efi number,deepdive,testbed,202210200116,test,irrelevant
5678,6158,"2022-10-19 07:57:24,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 07:57:24,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Datastore file operation] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:57:25,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Get specified property info for VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263'] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number data store file operation task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi para virtual vmxnet number get specified property info for vm u os server v number pvscsi vmxnet number efi number task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm u os server v number pvscsi vmxnet number efi number,deepdive,testbed,202210200116,machine,irrelevant
5678,6158,"2022-10-19 07:57:24,019 | Failed at Play [deploy_vm_efi_paravirtual_vmxnet3] *********
2022-10-19 07:57:24,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Datastore file operation] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/esxi_check_delete_datastore_file.yml:24
fatal: [localhost]: FAILED! => File 'OS/Linux/UOS/V20/1050/uniontechos-server-20-1050a-amd64.iso' is absent, cannot continue
2022-10-19 07:57:25,019 | TASK [deploy_vm_efi_paravirtual_vmxnet3][Get specified property info for VM 'UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263'] 
task path: /home/worker/workspace/Ansible_UOS_Server_V20_70GA_PARAVIRTUAL_VMXNET3_EFI/ansible-vsphere-gos-validation/common/vm_get_config.yml:4
fatal: [localhost]: FAILED! => Unable to gather information for non-existing VM UOS-Server-V20-1050a-PVSCSI_VMXNET3_EFI-HWv17_1666166176263",,timestamp failed at play deploy vm efi para virtual vmxnet number timestamp task deploy vm efi para virtual vmxnet number data store file operation task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common esxi check delete data store file yml number fatal localhost failed file os linux u os v number union tech os server number amd number i so is absent can not continue timestamp task deploy vm efi para virtual vmxnet number get specified property info for vm u os server v number pvscsi vmxnet number efi number task path home worker workspace ansible u os server v number ga para virtual vmxnet number efi ansible vsphere gos validation common vm get configuration yml number fatal localhost failed unable to gather information for non existing vm u os server v number pvscsi vmxnet number efi number,deepdive,testbed,202210200116,product,irrelevant
